deploy-manage/security/encrypt-deployment-with-customer-managed-encryption-key.md

```diff
@@ -24,7 +29,7 @@ Using a customer-managed key helps protect against threats related to the manage
 
 Using a customer-managed key can help comply with regulations or security requirements, but it is not a complete security solution by itself. There are other types of threats that it does not protect against.
 
-[1] You set up your customer-managed keys and their access in your key management service. When you provide a customer-managed key identifier to Elastic Cloud, we do not access or store the cryptographic material associated with that key. Customer-managed keys are not directly used to encrypt deployment or snapshot data. Elastic Cloud accesses your customer-managed keys to encrypt and decrypt data encryption keys, which, in turn, are used to encrypt the data.
+[1] You set up your customer-managed keys and their access in your key management service. When you provide a customer-managed key identifier to {{ecloud}}, we do not access or store the cryptographic material associated with that key. Customer-managed keys are not directly used to encrypt deployment or snapshot data. {{ecloud}} accesses your customer-managed keys to encrypt and decrypt data encryption keys, which, in turn, are used to encrypt the data.
```

Comment:
The `[1]` at the beginning of the line doesn't feel right, but maybe it's me that I don't understand it :)

Reply:
That's because there's a kind of note in the bullet points above, and because the note would be too disruptive, it's been handled this way 
<img width="617" alt="image" src="https://github.com/user-attachments/assets/2df4c4b2-4ab6-48d2-930c-6bbcd4ffa41d" />